backend/api/blueprints/leads.py

```diff
@@ -361,6 +365,45 @@ def _create_new_lead(request, valid_data_sources=VALID_DATA_SOURCES):
             if not all(c.isalnum() or c.isspace() or c == "." or c == "&" or c == "'" for c in company_name):
                 return {"message": "company_name can only contain a-z, 0-9, ., &, ', whitespace"}, 400
 
+    # using pydantic model to check phone number, assigned, email and social media urls
+    class FieldTestModel(BaseModel):
```

Comment:
Can you create a file called `models.py` in `api` and move this class there?

Reply:
On a second pass I realized that this class depends on functions in this module, so maybe leave it here for now to avoid making a cyclical dependency. We'll need to put thought into splitting this views file up to be more modular. Moving the database queries into a separate file is probably a prerequisite to separating out the models.